java/src/main/java/com/factset/protobuf/stach/extensions/models/TableData.java

```diff
@@ -4,10 +4,12 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import com.google.protobuf.Value;
 
 public class TableData {
     private List<Row> Rows = new ArrayList<Row>();
     private Map<String, String> metadata = new HashMap<String, String>();
+    private Map<String, List<Value>> rawMetadata = new HashMap<String, List<Value>>();
```

Comment:
shall we make this type as **Map<String, Value>** for rawMetadata ?
The Value type can have list of values by adding them to ListValue.
you can refer to this [snippet ](https://www.tabnine.com/code/java/methods/com.google.protobuf.ListValue/newBuilder?snippet=5ce6ad3537d9ff0004d5b427)for example

Reply:
I can definitely try that, but it was my understanding that rawMetadata should return `Map<String, List<Value>>`. @a-ghag what do you think?